src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.spec.jsx

```diff
@@ -1,25 +1,60 @@
 import { screen, fireEvent, waitFor } from '@testing-library/react';
 import { act } from 'react-test-renderer';
 import { renderWithTimecardContext } from '../../../test/helpers/TimecardContext';
-
 import SelectTimecardPeriodType from './SelectTimecardPeriodType';
 
 const mockRegister = jest.fn();
 const handleSubmit = jest.fn();
 const errors = {};
+const mockTimePeriodTypesResponse = {
+  data: {
+    meta: {
+      next: null,
+    },
+    items: [
+      {
+        name: 'Shift',
+      },
+      {
+        name: 'Scheduled rest day',
+      },
+      {
+        name: 'Non-working day',
+      },
+      {
+        name: 'On call',
+      },
+      {
+        name: 'Absence',
+      },
+      {
+        name: 'Training',
+      },
+      {
+        name: 'Overtime',
+      },
+    ],
+  },
+};
```

Comment:
Same here for importing `timeCardPeriodTypes` from mock data

Reply:
Good point. Incorporated